src/test/java/home18work.java

```diff
@@ -0,0 +1,45 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import javax.swing.*;
+
+public class home18work  extends BaseTest{
+
+    @Test
+public void playaSong()  {
+        login("karkisuman0330@gmail.com","te$t$tudent");
+        allSong();
+        choose();
+       equilizerDisplayed();
+    }
+        public void allSong() {
+            WebElement aSongBtn = driver.findElement(By.xpath("//a[@class=\"songs\"]"));
+            aSongBtn.click();
+
+    }
+       public static void choose() {
+        WebElement songName = driver.findElement(By.xpath("//section[@id=\"songsWrapper\"]//tr[3]"));
+        songName.click();
+        // i try doubleclick its not working comment plz
+           //Actions try1 =new Actions(driver);
+           //try1.doubleClick(songName);
+           WebElement playNextButton = driver.findElement(By.xpath("//i[@data-testid='play-next-btn']"));
+        WebElement playButton = driver.findElement(By.cssSelector("[data-testid='play-btn']"));
+        playNextButton.click();
+        playButton.click();
+
+
+    }
+
+
+
```

Comment:
**Please delete the empty line!**  An Empty Line is a “Code Smell” 

The short answer is, They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.